authentication.js

```diff
@@ -25,6 +25,8 @@ module.exports = {
   // The test method allows Zapier to verify that the credentials a user provides are valid. We'll execute this
   // method whenver a user connects their account for the first time.
   test: testAuth,
-  // assuming "username" is a key returned from the test
-  connectionLabel: '{{username}}'
+  // assuming "username" is a key in the json returned from testAuth
+  connectionLabel: (z, bundle) => {
+    return bundle.inputData.json.username;
+  }
```

Comment:
So, taking a look, I definitely agree that there's an error here (i.e. the current code doesn't work and will leave your label as `{{username}}`. 
However, I think there's a better fix for this issue. The problem lies in line 14 of this file – instead of returning the `response`, the `testAuth` function should be returning `response.json`. When that happens, you can do
```
connectionLabel: '{{username}}'
```
or
```
connectionLabel: (z, bundle) => {
  return bundle.inputData.username;
}
```